apps/studio/data/organizations/organizations-query.ts

```diff
@@ -1,16 +1,30 @@
 import { QueryClient, useQuery, UseQueryOptions } from '@tanstack/react-query'
+
+import { components } from 'api-types'
 import { get, handleError } from 'data/fetchers'
 import type { Organization, ResponseError } from 'types'
 import { organizationKeys } from './keys'
 
+function castOrganizationResponseToOrganization(
+  org: components['schemas']['OrganizationResponse']
+): Organization {
+  return {
+    ...org,
+    billing_email: org.billing_email ?? 'Unknown',
+    managed_by: org.slug.startsWith('vercel_icfg_') ? 'vercel-marketplace' : 'supabase',
+    partner_id: org.slug.startsWith('vercel_') ? org.slug.replace('vercel_', '') : undefined,
```

Comment:
just to double check - are these FE specific properties or API also returns managed_by and partner_id

Reply:
FE specific